_posts/2018-08-24-running-strimzi-on-aks.md

```diff
@@ -1,31 +1,31 @@
 ---
 layout: post
-title:  "Running Strimzi on Azure Container Service"
+title:  "Get Kafka on Azure Container Service in 15 minutes using Strimzi"
```

Comment:
"Get Apache Kafka ..." ?

Reply:
Actually how about: "Apache Kafka in the Cloud in 15 minutes using Strimzi and AKS".

Reply:
Better! Go for it!